ui/src/app/views/workspace/component.ts

```diff
@@ -323,22 +324,50 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     this.clusterService
         .localizeNotebook(this.workspace.namespace, this.workspace.id, fileList)
         .subscribe(() => {
-          this.alertCategory = 'alert-success';
-          this.alertMsg = 'File(s) have been saved';
-          this.showAlerts = true;
+          this.handleLocalizeSuccess();
           setTimeout(() => {
             this.resetAlerts();
           }, 5000);
         }, () => {
-          this.alertCategory = 'alert-danger';
-          this.alertMsg = 'There was an issue while saving file(s) please try again later';
-          this.showAlerts = true;
+          this.handleLocalizeError();
           setTimeout(() => {
             this.resetAlerts();
           }, 5000);
         });
   }
 
+  localizeAllFiles(): void {
+    this.errorHandlingService.retryApi(this.workspacesService
+        .retrieveAndLocalizeFiles(this.wsNamespace, this.wsId))
+        .subscribe(() => {
+              this.handleLocalizeSuccess();
+              setTimeout(() => {
+                this.resetAlerts();
+                this.clusterPulled = true;
+              }, 3000);
```

Comment:
What's this timeout for? It's not clear this will finish in 3 seconds, if that's what this means...